python/paddle/fluid/tests/unittests/mkldnn/test_elementwise_add_mkldnn_op.py

```diff
@@ -55,15 +55,25 @@ def init_input_output(self):
         self.y = np.random.uniform(1, 2, [4, 32]).astype(self.dtype)
         self.out = np.add(self.x, self.y)
 
-    # TODO(jczaja): Enable when grad is ready
     def test_check_grad_normal(self):
-        pass
+        self.check_grad(
+            ['X', 'Y'], 'Out', max_relative_error=0.05, check_dygraph=False)
 
     def test_check_grad_ingore_x(self):
-        pass
+        self.check_grad(
+            ['Y'],
+            'Out',
+            no_grad_set=set("X"),
+            max_relative_error=0.05,
```

Comment:
What is default `max_relative_error`? The error message says this part we'd better use default.

check_dygraph baidu will approve later because CPU part it is always False.

Reply:
Ok, I reverted changes in this file, as still we do not support broadcasting in backward. so there was fallback to CPU implementation anyway. I will bring back those tests when bradcasting in grad of elementwise_add is supported.